components/alert/alert.css

```diff
@@ -0,0 +1,29 @@
+.alert{
+    font-size: 1.2rem;
+    padding: 0.6rem;
+    max-width: 500px;
+    border-radius: 0.5rem;
+}
+.alert-info{
+    color:var(--dark-blue) ;
+    border: 1px solid var(--medium-blue) ;
+    background-color: var(--light-blue);
+    
+}
+.alert-success{
+    color:var(--dark-green) ;
+    border: 1px solid var(--medium-green)  ;
+    background-color: var(--light-green) ;
+    
+}
+.alert-error{
+    color: var(--dark-red);
+    border: 1px solid var(--medium-red);
+    background-color: var(--light-red);
```

Comment:
the only difference here is color and background color, you could have the border property in the alert itself.

Reply:
The border color for each alert is different , so I mentioned it specifically in each alert